.github/CHANGELOG.md

```diff
@@ -303,6 +303,10 @@ Josh Izaac
 
 <h3>Improvements</h3>
 
+* A new ``Wires`` class was introduced for the internal 
```

Comment:
```suggestion
* A new `Wires` class was introduced for the internal 
```

Reply:
I apologize profoundly, I am so confused between rst, markdown, slack and so on, these mistakes just sneak in.

Reply:
No stress, everyone is doing it 😆 It's an historical accident that our changelog is in markdown, while everything else is in rst. I even had to remove some `.. code-block::`'s from the changelog before release.